shared-libs/phone-number/test/libphonenumber.spec.js

```diff
@@ -22,210 +22,237 @@ const settingsWithNoPhoneValidation = {
 
 describe('libphonenumber', () => {
 
-  it('normalize does nothing when already normalized', () => {
```

Comment:
Sorry about this massive diff... I wrapped each section in a `describe` block so that meant indenting each line.